content/ja/tracing/trace_collection/dd_libraries/dotnet-core.md

```diff
@@ -95,7 +95,14 @@ Datadog の .NET Framework ライブラリとプロセッサアーキテクチ
 {{< partial name="apm/apm-containers.html" >}}
 </br>
 
-3. {{< site-region region="us3,us5,eu,gov" >}}
+3. The tracing client attempts to send traces to the following:
+
+- The `/var/run/datadog/apm.socket` Unix domain socket by default.
+- If the socket does not exist, then traces are sent to `localhost:8126`.
+- If a different socket, host, or port is required, use the `DD_TRACE_AGENT_URL` environment variable: `DD_TRACE_AGENT_URL=http://custom-hostname:1234` or `DD_TRACE_AGENT_URL=unix:///var/run/datadog/apm.socket`
+- Using Unix Domain Sockets for trace transport is only supported on .NET Core 3.1 and later.
+
+For more information on how to configure these settings, see [Configuration](#configuration).
```

Comment:
Where did this content come from?

Reply:
@kayayarai 

This was copied from the English equivalent file. Its not common to have english in the ja so perhaps this was a bad move? 

The problem was this site region shortcode shouldn't be here in order for hugo to build. It seemed weird to remove it and have it go `1.` , `2.` and then `4.` so i filled `3.` with whats in the english. Do you think its better to just remove it instead?